app/views/admins/_form.html.slim

```diff
@@ -1,15 +1,12 @@
-= form_for @admin do |f|
-  - if @admin.errors.any?
-    #error_explanation
-      h2 = "#{pluralize(@admin.errors.count, "error")} prohibited this admin from being saved:"
-      ul
-        - @admin.errors.full_messages.each do |message|
-          li = message
-
-  .field
+= form_with model: admin, local: true do |f|
+  = render partial: 'shared/error', locals: { model: admin }
+  p
     = f.label :email
+    br
     = f.text_field :email
-  .field
+  p
```

Comment:
nah p tag isnt right
a label and its input is not a paragraph
```
The HTML <p> element represents a paragraph. Paragraphs are usually represented in visual media as blocks of text separated from adjacent blocks by blank lines and/or first-line indentation, but HTML paragraphs can be any structural grouping of related
```

Reply:
go look up the common tags for html and get more familiar with them when you can since this should have felt wrong

Reply:
the other forms for the other models use p tags tho, should i change it back to the `.field`?

Reply:
was the `.field` div doing any kind of styling or was it just a div?

Reply:
> was the `.field` div doing any kind of styling or was it just a div?

its just a div with `.field` class

Reply:
sure then this should be fine